src/util/nextThought.js

```diff
@@ -34,8 +34,7 @@ import { equalArrays } from './equalArrays.js'
  */
 const isValidContextView = path =>
   path.length &&
-  isContextViewActive(pathToContext(path)) &&
-  getContexts(head(path).value).length > 1
+  isContextViewActive(pathToContext(path))
```

Comment:
Since this has been simplified, perhaps we can use `isContextViewActive` directly and avoid the extra function definition.